### DISCUSSION TITLE
Link missing in 4.6 announcement blog?

### DISCUSSION THREAD

**Discussion by DierkDroth** [unoplatform/uno#10335] [Q&A] [answered]
Hi,

I was reviewing the [4.6 announcement](https://platform.uno/blog/uno-platform-4-6-hello-net-7/) and was trying to locate this: "For a detailed list of over 200 PRs, please head to our release notes on GitHub".

Where in the [doc section](https://github.com/unoplatform/uno/tree/master/doc) would I find the release notes?

Thanks

**Comment by jeromelaban** [comment, marked as answer]
Thanks for the report. Releases are always located in the release section from the main repo page. See [here](https://github.com/unoplatform/uno/releases) for a direct link.

**Reply by DierkDroth** [reply]
Thanks @jeromelaban
